packages/manager/modules/anthos/src/dashboard/general-information/general-information.html

```diff
@@ -138,9 +163,16 @@
             >
                 <!--Hosts-->
                 <oui-tile-definition
-                    data-term="{{:: 'anthos_tenant_dashboard_general_information_tile_resources_hosts' | translate }}">
-                    <oui-action-menu aria-label="Actions" data-compact data-placement="end">
-                        <oui-action-menu-item data-on-click="$ctrl.onGoToOrderHost()">
+                    data-term="{{:: 'anthos_tenant_dashboard_general_information_tile_resources_hosts' | translate }}"
+                >
+                    <oui-action-menu
+                        aria-label="Actions"
```

Comment:
**nitpick:** Might be great if we can move this to a translation file.

Reply:
This is done